pages/services/dcos-monitoring/1.0.0/operations/alertmanager/storage/index.md

```diff
@@ -12,13 +12,13 @@ model: ../../../data.yml
 
 ## DSS Volume
 
-You can configure Alertmanager to use a dedicated volume provided by the [DC/OS Storage Service](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/) (DSS).
+You can configure Alertmanager to use a dedicated volume provided by the [DC/OS Storage Service](https://docs.mesosphere.com/services/storage/1.0.0/) (DSS).
```

Comment:
Wonder whether we should just point to `/storage/latest` here

Reply:
@orlandohohmeier , you can, we have a 307 redirect set up for storage latest